test/integration/rules/empty-table-header/empty-table-header.json

```
@@ -0,0 +1,6 @@
+{
+  "description": "empty-table-header tests",
+  "rule": "empty-table-header",
+  "incomplete": [["incomplete1"], ["incomplete2"], ["incomplete3"]],
```

Almost there! Just need to add the id selector `#` to each of these for the tests to pass.
```suggestion
  "incomplete": [["#incomplete1"], ["#incomplete2"], ["#incomplete3"]],
```